webapp/lib/controller.dart

```diff
@@ -99,31 +102,36 @@ void command(UIAction action, Data data) {
 
     /*** Data */
     case UIAction.needsReplyDataUpdated:
-      Map<DateTime, int> data = new Map.fromIterable(needsReplyDataList,
+      view.contentView.projectSelectorView.populateProjects(projectList);
+
+      var selectedProjectNeedsReplyDataList = needsReplyDataList.where((
+        d) => d.project == view.contentView.projectSelectorView.selectedProject).toList();
```

Comment:
nit: weird line break - can you change it so you have `(d) => ...` and indented 4 spaces from the row above? thanks

Reply:
Fixed that